TranslucentTB/config.cfg

```diff
@@ -0,0 +1,3 @@
+accent=transparent
```

Comment:
I would prefer the default config to be blur, as this has been the default state.